metadata_service/config.py

```diff
@@ -58,7 +58,30 @@ class Config:
     STATISTICS_FORMAT_SPEC: Dict[str, Dict] = {}
 
     # whitelist badges
-    WHITELIST_BADGES: List[str] = []
+    WHITELIST_BADGES: List[Dict] = [{'badge_name': 'alpha',
+                                     'category': 'table_status',
+                                     'badge_type': 'neutral'},
+                                    {'badge_name': 'beta',
+                                     'category': 'table_status',
+                                     'badge_type': 'neutral'},
+                                    {'badge_name': 'GA',
+                                     'category': 'table_status',
+                                     'badge_type': 'positive'},
+                                    {'badge_name': 'deprecated',
+                                     'category': 'table_status',
+                                     'badge_type': 'negative'},
+                                    {'badge_name': 'recommended',
+                                     'category': 'recommended',
+                                     'badge_type': 'positive'},
+                                    {'badge_name': 'Core Concepts',
```

Comment:
I think we put this config into our private and leave it open here. All these badges defined (e.g coco) are Lyft specific.